src/core/cgroup.h

```diff
@@ -474,9 +442,6 @@ uint64_t unit_get_ancestor_memory_low(Unit *u);
 uint64_t unit_get_ancestor_startup_memory_low(Unit *u);
 
 int unit_search_main_pid(Unit *u, PidRef *ret);
-int unit_watch_all_pids(Unit *u);
-
-int unit_synthesize_cgroup_empty_event(Unit *u);
```

Comment:
(I have this in my queue already, with some extra changes)

Reply:
OK. Do you want to drop the commit? Or should I pick your commit in your branch?

Reply:
I'll submit them later.

BTW, I'm going to give basic/cgroup-util.[ch] a decent rework, so maybe leave that when you work on cleanups?

Reply:
Sure.